src/redis_cmd.cc

```diff
@@ -4714,9 +4714,7 @@ class CommandScript : public Commander {
 };
 
 #define ADD_CMD(name, arity, description , first_key, last_key, key_step, fn) \
-{name, arity, description, 0, first_key, last_key, key_step, []() -> std::unique_ptr<Commander> { \
-  return std::unique_ptr<Commander>(new fn()); \
-}}
+{name, arity, description, 0, first_key, last_key, key_step}
```

Comment:
Why did you remove `fn`? I think this is a classic factory pattern.

Reply:
The role of the factory is to dispatch from the command name to the corresponding derived class of `Commander`